### ISSUE TITLE
Custom Facade Helper Generation

### ISSUE THREAD

**Issue by andrey-helldar** [barryvdh/laravel-ide-helper#988]
Now the package can generate helpers for facade methods only if they are written to the configuration file.

Therefore, yesterday I [published a package](https://github.com/andrey-helldar/laravel-ide-facades-helper) that generates them from the heirs of the main facade found in the project.

I want to ask you. Will your package support this feature or:
1. should I continue to support my package?
2. you can send PR to add this functionality (or I can).

**Comment by mfn** [comment]
> that generates them from the heirs of the main facade found in the project

Maybe I'm missing something, but wouldn't that mean that it generates code for something not enabled in the first place? 🤔

**Comment by andrey-helldar** [comment]
Google translated so that I did not understand the essence.Now the ide-helper package from Barry does not know how to generate files for these facades. So I created a package. Now I think it is not easier to send the code to the Barry package... 🤔Best Regards    Andrey Helldar18:45, 9 июля 2020 г., Markus Podar <notifications@github.com>:

that generates them from the heirs of the main facade found in the project

Maybe I'm missing something, but wouldn't that mean that it generates code for something not enabled in the first place? 🤔
